__tests__/authmware.spec.js

```diff
@@ -0,0 +1,57 @@
+//
+//
+// Copyright © 2018 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Jason Leach on 2018-11-05.
+//
+
+/* eslint-env es6 */
+
+'use strict';
+
+import fs from 'fs';
+import path from 'path';
+import { verify } from '../src/libs/authmware';
+
+const path0 = path.join(__dirname, 'fixtures/jwt-decoded-norole-20181105.json');
+const decodedNoRole = JSON.parse(fs.readFileSync(path0, 'utf8'));
+
+const path1 = path.join(__dirname, 'fixtures/jwt-decoded-roleok-20181105.json');
+const decodedWithRole = JSON.parse(fs.readFileSync(path1, 'utf8'));
+
+describe('Test Authentication', () => {
```

Comment:
could be nice if including the case where no `jwtPayload` is passed in `verify` as well

Reply:
Done.